CHANGELOG.rst

```diff
@@ -1,4 +1,7 @@
 We don't maintain a detailed changelog.  For details of changes, please see
 the `GitHub commit history`_.
 
+ - 2.4.6
```

Comment:
🎉  Nice, we can come up with some instructions and a format for maintaining this file but good to at least get started.